src/components/DatGui.vue

```diff
@@ -135,6 +135,11 @@ const handleClick = () => {
   .control-item {
     border-left: var(--border-left-size) solid;
 
+    &.disabled {
+      background-color: gray;
+      opacity: 0.5;
+    }
```

Comment:
Maybe add a `pointer-events: none;` so we cannot interact with any children on hover.

```suggestion
    &.disabled {
      background-color: gray;
      opacity: 0.5;
      pointer-events: none;
    }
```